### ISSUE TITLE
Instructions on training

### ISSUE THREAD

**Issue by lizhe00** [zju3dv/RelightableAvatar#1]
Hi, nice work! Could you please tell me when to provide instructions on training?

**Comment by HelloRicky123** [comment]
Same question. Do you have plan for the two TODO list? @lizhe00

**Comment by HelloRicky123** [comment]
Hi Sida, I have the same question. Do you have plan for the two TODO list recently? @pengsida

**Comment by pengsida** [comment]
Hi, do you need it urgently?

**Comment by HelloRicky123** [comment]
Actually yes, a few days ago, I intended to incorporate this code into my experiment and submit it to Siggraph Asia. However, I encountered issues during the training process and when using the previous SyntheticHuman dataset to evaluate the lighting effects, resulting in a failed submission. Now, I aim to continue to enhance the experiment using this test data and evaluation criteria.

**Comment by pengsida** [comment]
Our code for the training stage has already been open-sourced, but we haven't written the documents yet. We will find time to write them in the next few weeks.

**Comment by HelloRicky123** [comment]
Yes, I have tested the training code and found it to be beneficial. I am looking forward to receiving the document. Could you please share the SyntheticHuman++ dataset with me? Additionally, I would like the file for evaluating Albedo and Relighting in the paper.

Thanks a lot!

**Comment by pengsida** [comment]
Here are the [Mobile-Stage dataset](https://docs.google.com/forms/d/e/1FAIpQLSeEbjuTV7w0lfryl-9FPX1VteuPGkbqjDvxXebY02Tm6BMejQ/viewform?usp=sf_link) and [SyntheticHuman++ dataset](https://docs.google.com/forms/d/e/1FAIpQLSctrS3NZ5pThqQzakvitykNgJxpGifT9AOKEtWYZ_JqFQIb3w/viewform?usp=sf_link).

**Comment by HelloRicky123** [comment]
Received. Thanks a lot again!

**Comment by Jiatttliu** [comment]
Hi, @pengsida @HelloRicky123 . Thanks for your share, but I can not unzip the SyntheticHuman++ dataset correctly, could you provide the instruction?
